events/events.go

```diff
@@ -78,7 +78,7 @@ func (p *publisher) Get() interface{} {
 }
 
 func (p *publisher) GetAndSubscribe() (interface{}, Subscriber, error) {
-	c := make(chan struct{})
+	c := make(chan struct{}, 2)
```

Comment:
This is so that an notification is not missed in the case of you receive a notification but before you unblock and read it another is sent and is dropped when enqueuing.